stmhal/usbd_cdc_interface.c

```diff
@@ -45,38 +45,37 @@
 #include "irq.h"
 #include "timer.h"
 #include "usb.h"
-
-// CDC control commands
-#define CDC_SEND_ENCAPSULATED_COMMAND               0x00
-#define CDC_GET_ENCAPSULATED_RESPONSE               0x01
-#define CDC_SET_COMM_FEATURE                        0x02
-#define CDC_GET_COMM_FEATURE                        0x03
-#define CDC_CLEAR_COMM_FEATURE                      0x04
-#define CDC_SET_LINE_CODING                         0x20
-#define CDC_GET_LINE_CODING                         0x21
-#define CDC_SET_CONTROL_LINE_STATE                  0x22
-#define CDC_SEND_BREAK                              0x23
```

Comment:
Yes this is not related I agree.

The original #defines by ST are placed in the usbd_cdc.h so I found it odd that they are again defined in the cdc's *.c file... I haven't moved anything to the header, it was already there.

But I can include it again no problem